### PR TITLE
[TD] Command.cpp: uniform menu names

### PR DESCRIPTION
**Pull request by donovaly**
see https://forum.freecadweb.org/viewtopic.php?f=35&t=40608 for the reason

- also some simplifications

### CODE REVIEW

**Comment by donovaly** [comment]
PR https://github.com/FreeCAD/FreeCAD/pull/2684 should be merged wit this one because otherwise the changed menu name for balloon makes no sense.

File: src/Mod/TechDraw/Gui/Command.cpp [hunk 1]
```diff
@@ -94,8 +94,8 @@ CmdTechDrawNewPageDef::CmdTechDrawNewPageDef()
 {
     sAppModule      = "TechDraw";
     sGroup          = QT_TR_NOOP("TechDraw");
-    sMenuText       = QT_TR_NOOP("Insert new default Page");
-    sToolTipText    = QT_TR_NOOP("Insert new default Page");
+    sMenuText       = QT_TR_NOOP("Insert new default page");
+	sToolTipText    = sMenuText;
```
Comment (vocx-fc) [inline src/Mod/TechDraw/Gui/Command.cpp]: It seems you accidentally added a Tab as indentation. I would keep the style of the previous code and use only spaces.

Comment (donovaly) [reply src/Mod/TechDraw/Gui/Command.cpp]: Thanks. this is now fixed.

**Comment by WandererFan** [comment]
FEM unit test errors in Win build.  Ignored and merged.

**Pull request merged**